pom.xml

```diff
@@ -58,6 +58,9 @@
 
   <properties>
     <license.title>SonarQube IaC Plugin</license.title>
+    <sonar.organization>sonarsource</sonar.organization>
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
```

Comment:
It's strange that we have to repeat that, it kinds of duplicate `SONAR_HOST_URL`.